assembly/src/assembler/mod.rs

```diff
@@ -767,65 +766,76 @@ impl Assembler {
                 Op::If {
                     then_blk, else_blk, ..
                 } => {
-                    if let Some(basic_block_id) = basic_block_builder.make_basic_block(mast_forest)
+                    if let Some(basic_block_id) =
+                        basic_block_builder.make_basic_block(mast_forest_builder)
                     {
                         mast_node_ids.push(basic_block_id);
                     }
 
                     let then_blk =
-                        self.compile_body(then_blk.iter(), context, None, mast_forest)?;
-                    let else_blk =
-                        self.compile_body(else_blk.iter(), context, None, mast_forest)?;
+                        self.compile_body(then_blk.iter(), context, None, mast_forest_builder)?;
```

Comment:
Looks like there was a merge conflict here - the old version in the diff is actually newer than the "new" version in the diff: we no longer treat the `else` block specially here (either block could be empty now, and we rely on `compile_body` to fill empty bodies with a single `nop`, if any have made it to the assembler).

Reply:
fixed in [3c26bd6](https://github.com/0xPolygonMiden/miden-vm/pull/1359/commits/3c26bd662ca26306608e7519f8875a002764a3bb)